src/electron/platform/android/scan-results.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export class ScanResults {
+    private rawData: any;
+
+    constructor(rawData: any) {
+        this.rawData = rawData;
+    }
+
+    public get deviceName(): string {
+        try {
+            return this.rawData.axeContext.axeDevice.name;
+        } catch {
+            return null;
```

Comment:
I chose to return null here since it seemed like a bad idea to leak exceptions caused by the private implementation. Happy to throw here if that's the general preference